nipype/interfaces/fsl/model.py

```diff
@@ -1134,28 +1134,28 @@ class L2Model(BaseInterface):
 
     def _run_interface(self, runtime):
         cwd = os.getcwd()
-        mat_txt = ['/NumWaves       1',
-                   '/NumPoints      %d' % self.inputs.num_copes,
-                   '/PPheights      %e' % 1,
+        mat_txt = ['/NumWaves   1',
+                   '/NumPoints  {}'.format(self.inputs.num_copes),
+                   '/PPheights  1',
```

Comment:
I believe that the original code was based on replicating the files that are produced by FEAT as closely as possible. It's possible these changes are benign or even better justified, but I don't like seeing changes to functionality without a justification (especially since any problems that might be introduced here aren't going to be covered by the unit tests).